### ISSUE TITLE
Reviewed Files [nui.zip]

### ISSUE THREAD

**Issue by sabeena21** [Samsung/tizen-docs#208]
Hello Catherine,

Please find the reviewed file in the location:
https://github.com/Samsung/tizen-docs/files/1784131/nui.zip)
Please do let me know if you can access the path.

Regards,
Sabeena

**Comment by mijin85cho** [comment]
To  record the history, I copy the mail thread of this update.


**[animation.md]**
- line 14 : We have "Properties" section, while we have "Animation Properties" section right below. Aren't they similar overlapped sections?
**sabeena: I can delete this section to avoid conflict.**

- line 40 : To make a list, please use - instead of * in front of each list items
**sabeena - Sure, I will make changes**

- line 85 and below : To make a list, please use - instead of * in front of each list items
**sabeena - sure, I will make changes**

- line 95 : I can't find animation.SetLooing() on https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.Animation.html#methods. Would you let me know from where did you get it?
**sabeena: I was comparing the section in animation file present  in Dotnet with the animation file present in the Native Application, comparing these two files,  I had replaced "animation.Looping = true" to " animation.SetLooping(true)".**
**If it is incorrect, I will again change it to original**



**[animation-types.md and animation - types.md]**

You have duplicated files, animation-types.md(original) and animation - types.md (copy version, I guess)
**Sabeena : I can delete the  duplicate file**



**[layer.md]**

- line 13 : Layers provides => I'm not sure, but isn't "Layers provide" correct?
**Sabeena: It is correct, I can accept the change**


**[resources.md]**

- line 17 : "url" is changed into URL before, and now it is "url" again. Would you please confirm it?
**Sabeena : According to our MSTP style guide it is URL. We refer this guide to review the content.**

- line 18 : The space before : is removed before, and reverted here. Would you please  confirm it?
**Sabeena: Yes correct, It is good to avoid whitespace before colon.**

- line 72 : This line is feedbacked as redundant and reverted here. Would you please  confirm it?
**Sabeena: Correct**

- line 76 : `FittingMode` is changed into "fitting modes" before, and reverted here. Would you please confirm it?
**Sabeena: correct**

- line 78 : The sentences in the table looks reverted to old version. Would you please confirm it?
**SAbeena: correct**



[styling-controls-with-JSON.md]

- line 157 : "2" is converted into "three". Would you please check?
**Sabeena: Correct the changes from 2 to two not three. Sorry,for this**


**[texteditor.md]**

- line 69 and line 84 : There should be no empty lines in the table. The empty line wil break the rest of the table.
**Sabeena: I accept the changes**

- line 83 : In side of a table, we should use <br> to make another line. (ex. Enable the Markup string, to process text within the Markup tags using DALi application.<br>**Note**: By default, the Markup string is disabled.)
**SAbeena: Accept the changes**


**[textfield.md]**

- line 56 : "If there no enough space available for the text displayed." Aren't we missing a verb in this sentence..?
**sabeena :Rephrased the sentence**

- line 92 : In line 69 of the texteditor.md, we have "of the cursor" at the end of the sentence for same "CursorBlinkInterval" How about removing it also?
**Sabeena: accept the change**

- line 97 : In line 64 of the texteditor.md, that sentence is using "occurs" as a verb while this sentence is using "will occur". wouldn't it better to match the tones between the sentences?
**Sabeena: accept the change**

- line 106 : "with the on-screen area" It looks like "within" is correct. Would you please check?
**Sabeena: accept the changes**

- line 107 : "The settings to related to the System's ...". I'm not sure, but isn't "The setting related to the Ssytem's ..." better? 
**SAbeena: accept the changes**

- line 110 :  In side of a table, we should use <br> to make another line. (ex. Enable the Markup string, to process text within the Markup tags using DALi application.<br>**Note**: By default, the Markup string is disabled.)
**sabeena: accept the changes**

- line 126 : "ellipses" is used here, while in line 181 for textlabel.md "ellipsis" is used.
**sabeena: accept the changes**

**[textlabel.md]**

- line 159, and 160 : Don't we need a period at the end?
**sabeena: Added the changes**

- line 168 : "Underline height" is converted into "The Underline color of the height." I'm not sure about this change.
**sabeena: Added the changes**

- line 170 : In side of a table, we should use <br> to make another line. (ex. Enable the Markup string, to process text within the Markup tags using DALi application.<br>**Note**: By default, the Markup string is disabled.)
**sabeena: added the changes**

- line 180 : The tone of the sentence is note same with line 122 of textfield.md. "The font size is in pixels." and "The size of font in pixels."
**sabeena: added the changes**

**[videoview.md]**

- line 3 : "The `VideoView` class is a controls video playback and display." I'm not sure about this sentence. I'm confused with "a contorls"
**Sabeena:  I have done changes**

- line 26 and 62 : It looks like it would be ok to left as "VideView". Should we change into "Videoview" and "Video view"?
**SAbeena: I can keep the original "VideView"**

**[visuals.md]**

- line 3 : "Visuals are the main building block any UI components." I'm not sure about this sentence. Is it ok to use any instead of for?
**sabeena: Yes it be changes**

- line 9 : To make a list, please use - instead of * in front of each list items
**sabeena: Accepted changes**

- line 57 : Isn't it better to make a note in below format? I applied below format for now, so if it should be reverted, please let me know
```
    > **Note**  
    > The recently registered visual is mostly always on top.
```
**Yes it is**

- line 178 : "Whether border anti aliasing of the border is required." It looks like "border" is duplicated. Is it ok?
**sAbeena: Removed**

- line 322, 385, 393, and 499 : To make a list, please use - instead of * in front of each list items
**sabeena: done**

- line 439 : To make a list, please use - instead of * in front of each list items
**sabeena: done**



------------------------
The updated version for this review file is uploaded on another issue, #210.
Sabeena, to prevent scattering the issue articles of one PR, please use one thread for a PR from next time.

**Comment by mijin85cho** [comment]
Issue closed by #189.
